docs/source/conf.py

```diff
@@ -1,78 +1,48 @@
 # ----------------------------------------------------------------------------
-#  --- MUST ADAPT THIS FILE FOR YOUR PROJECT ---
+# --- MUST ADAPT THIS FILE FOR YOUR PROJECT ---
 # --- Instructions below ---
 #
 # * Search and replace `jupyterhub-python-repo-template` with the name of your
 #   repository
-# * Search after the `TODO` keyword and follow the instructions in the comment
 # ----------------------------------------------------------------------------
 
-# Configuration file for the Sphinx documentation builder.
-#
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
-
-# -- Path setup --------------------------------------------------------------
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
+# Configuration file for Sphinx to build our documentation to HTML.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
-
-# -- Project specific imports ------------------------------------------------
-
 import datetime
 
 # -- Project information -----------------------------------------------------
-
+# ref: https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+#
 project = "jupyterhub-python-repo-template"
 copyright = f"{datetime.date.today().year}, Project Jupyter Contributors"
 author = "Project Jupyter Contributors"
 
 
-# -- General configuration ---------------------------------------------------
+# -- General Sphinx configuration ---------------------------------------------------
 # ref: https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
-
-# Set the default role so we can use `foo` instead of ``foo``
-default_role = "literal"
```

Comment:
Was this removed because it's defaulting to this value anyway, or is there other reason?

Reply:
Nvm, I've seen it moved below! ✨ 🎉